src/applications/claims-status/utils/helpers.js

```diff
@@ -148,12 +177,14 @@ export function displayFileSize(size) {
 }
 
 export function groupClaimsByDocsNeeded(list) {
-  const claimsWithOpenRequests = list.filter(
-    claim => claim.attributes.documentsNeeded,
-  );
+  const groupingPredicate = c => {
+    return c.attributes.documentsNeeded && isInEvidenceGathering(c);
+  };
+
+  const claimsWithOpenRequests = list.filter(claim => groupingPredicate(claim));
```

Comment:
This is pedantic, but you can just do this:
`const claimsWithOpenRequests = list.filter(groupingPredicate);`